.github/workflows/qa.yml

```diff
@@ -10,17 +10,16 @@ jobs:
   test:
     name: linting & spelling
     runs-on: ubuntu-latest
-
     env:
       TERM: xterm-256color
 
     steps:
       - name: Checkout Code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Set up Python '3,11'
-        uses: actions/setup-python@v3
-        with:
+        uses: actions/setup-python@v5
+        with
```

Comment:
This introduces a syntax error which causes the `QA / linting & spelling` action to fail.